elastic/kibana
# [Fleet] Fix navigation from multi page layout

## Summary

Resolve #158212

Fix displaying the add agent flyout when coming from the multi page layout package policy editor.

## How to test /reproduce the initial bug 

1. Go to the multi page layout http://localhost:5601/app/fleet/integrations/nginx-1.15.0/add-integration?useMultiPageLayout=true 
2. click on add the integration without agent in the bottom bar
3. Create the package policy
4. Click on add agent => you should be redirected on the agent policy page with the enrollment flyout open.
